Update README.md (#729)
xamarin/XamarinCommunityToolkit: README.md

## Build Status

If you like to live dangerously, you can use our [nightly](https://pkgs.dev.azure.com/xamarin/public/_packaging/XamarinCommunityToolkitNightly/nuget/v3/index.json) feed to try out the latest and greatest.

| Build Server | Type         | Platform | Status |
|--------------|--------------|----------|--------|
| Azure DevOps         | Build        | Windows & Mac  | [![Build Status](https://dev.azure.com/xamarin/public/_apis/build/status/xamarin/CommunityToolkit/xamarin.XamarinCommunityToolkit%20(Public)?branchName=main)](https://dev.azure.com/xamarin/public/_build?definitionId=55&_a=summary) |

## Sample App

Browsing the [sample app](./samples) is the best place to start exploring what's available today.

## Installation

> The Toolkit is available via NuGet, and should be installed into all (shared and each platform) your projects:

Reviewer: > and should be installed into all (shared and each platform) your projects

"and should be installed into all of your projects (shared and individual platforms)"

> The Toolkit is available via NuGet, and should be installed into all (shared and each platform) your projects:

Reviewer: ```suggestion
and should be installed into all of your projects (shared and individual platforms)
```

> The Toolkit is available via NuGet, and should be installed into all (shared and each platform) your projects:

Reviewer: ```suggestion
The Toolkit is available via NuGet, and should be installed into all of your projects (shared and individual platforms):
```
